flink-connectors/flink-connector-jdbc/src/main/java/org/apache/flink/connector/jdbc/table/JdbcDynamicTableFactory.java

```diff
@@ -80,6 +80,11 @@
 		.noDefaultValue()
 		.withDescription("the class name of the JDBC driver to use to connect to this URL. " +
 			"If not set, it will automatically be derived from the URL.");
+	public static final ConfigOption<Duration> MAX_RETRY_TIMEOUT = ConfigOptions
+		.key("max-retry-timeout")
```

Comment:
Would be better to use `connection.max-retry-timeout` to make it clear that this is a timeout config for connection.  This also keeps align with Elasticsearch `connection.max-retry-timeout` config.